recipes/CoolProp/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "CoolProp" %}
+{% set version = "6.0.0" %}
+{% set sha256 = "f3c8b57a2f5cd6dc4816623a460defe3bdc0639bd1f709c2cfeb5776486d4ba3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```

Comment:
Is `cython` needed?

Reply:
I added a thread on their mailing-list:
https://groups.google.com/forum/#!topic/coolprop-users/oKDLMXn14RI

I'll wait for a reply to correctly answer you.

Thank you @synapticarbors

Reply:
I investigated this by myself and I realized that it uses cython if possible, but it is not required.
So, answering your question: no, it does not depend on cython.

Reply:
I think this piece of code answer you question:

https://github.com/CoolProp/CoolProp/blob/master/wrappers/Python/setup.py#L193

Besides that, c++ files which was auto-generated by Cython are in the tar.gz file from pypi.